marker_rustc_driver/src/context/map.rs

```diff
@@ -0,0 +1,50 @@
+use marker_adapter::context::AstMapDriver;
+use marker_api::{
+    ast::item::{EnumVariant, Field},
+    lint::Level,
+    prelude::*,
+};
+
+use super::RustcContext;
+
+impl<'ast, 'tcx: 'ast> AstMapDriver<'ast> for RustcContext<'ast, 'tcx> {
+    fn item(&'ast self, id: ItemId) -> Option<ItemKind<'ast>> {
+        let rustc_id = self.rustc_converter.to_item_id(id);
+        self.marker_converter.item(rustc_id)
+    }
+
+    fn variant(&'ast self, _id: VariantId) -> Option<&'ast EnumVariant<'ast>> {
+        todo!()
```

Comment:
I wonder how the panic across an FFI boundary manifests itself today. Is it going to be debuggable if this todo-ed code is mistakenly invoked during development?

Reply:
`panic!` calls across FFI boundaries are undefined behavior, in practice they trace to the FFI boundary and then abort with the stack trace until that point. This is something which should be addressed by Marker at some point #10 

Marker has two tests that verify the current behavior, one tracing from the lint crate to Marker and one from the driver. I have to check, why I left this one blank :thinking: